src/managed/CommonManaged.props

```diff
@@ -6,8 +6,14 @@
   <PropertyGroup>
     <RepoRoot>$([System.IO.Path]::GetFullPath('$(MSBuildThisFileDirectory)../..'))/</RepoRoot>
     <VersionPrefix>2.0.0</VersionPrefix>
```

Comment:
Gaurav's new 2.1 change points out a problem with this property.  We should be sharing the root dir.props values.

This points out one of the problems with dumping everything into dir.props files.  Here we just want to share the repo's "branch info".  We can't use all the other settings in the dir.props because it doesn't jive with the .NET Core SDK.

I've logged [https://github.com/dotnet/core-setup/issues/2435](https://github.com/dotnet/core-setup/issues/2435) for this.